dots.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>should this be bold? or moved into the next cell as a #comment? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>moved it</p>